prettier.config.js

```diff
@@ -0,0 +1,12 @@
+/* eslint-env node */
+module.exports = {
```

Comment:
Is there by any chance a way to reuse the same prettier configuration in all our packages? it would be handy in case we need to make updates

Reply:
Looks like we could do something like this https://prettier.io/docs/en/configuration.html#sharing-configurations

We should